src/Components/Aside/Aside.jsx

```diff
@@ -0,0 +1,19 @@
+import data from "../../stretched-goal.json"
+import { PlaylistItem } from "../PlaylistItem/PlaylistItem"
```

Comment:
I would love more comments in this file, just saying what aside.jsx does

Reply:
I will add some more comments. I tend to avoid adding comments, as writing English requires me to think. 
I will try my best to get used to it!